pkg/network/dhcp/server/conn.go

```diff
@@ -16,7 +16,7 @@
  * Copyright 2020 Red Hat, Inc.
  *
  */
-package dhcpv6
+package server
```

Comment:
There is code here which is DHCPv6 specific but the names do not represent it.
If there is a need to separate dhcp and dhcpv6, this is a good reason for it (context).

Reply:
What do you think about moving the IPv6 server to serverv6 ?